site/en/r2/tutorials/eager/custom_training.ipynb

```diff
@@ -272,7 +273,7 @@
       "source": [
         "### Obtain training data\n",
         "\n",
-        "Let's synthesize the training data with some noise."
+        "First, you can synthesize the training data by adding random Gaussian (Normal) noise to the inputs, which can make your model more robust against overfitting and thus aid generalization:"
```

Comment:
Delete this line:

"which can make your model more robust against overfitting and thus aid generalization:"

This is not here for regularization, it is here to simulate noisy real-world data. They're different things and this will only confuse people.

Reply:
Got it! Done. Thanks @MarkDaoust